src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // перетворити рядок на масив де елемент це правило css
+  // перевірити чи рядок не порожній
+
+  const listStyles = sourceString
+    .split(';')
+    .filter((string) => string.trim() !== '');
```

Comment:
The filter callback can be simplified to a single expression without the need for curly braces and an explicit return statement.